### PR TITLE
Fix the symbolic for pixel shuffle

### PR DESCRIPTION
**Pull request by houseroad**
Using Transpose + Reshape, not using DepthToSpace, since they are not available in C2 yet.

### CODE REVIEW

File: test/onnx/test_pytorch_onnx_caffe2.py [hunk 1]
```diff
@@ -828,6 +828,13 @@ def test_instance_norm(self):
         underlying = nn.InstanceNorm2d(3)
         self.run_model_test(underlying, train=False, batch_size=BATCH_SIZE)
 
+    def test_pixel_shuffle(self):
+        underlying = nn.PixelShuffle(4)
+        shape = (1, 64, 5, 5)
+        input = Variable(torch.randn(*shape),
+                         requires_grad=True)
+        self.run_model_test(underlying, train=False, input=(input), batch_size=BATCH_SIZE)
```
Comment (BIT-silence) [inline test/onnx/test_pytorch_onnx_caffe2.py]: Line over 80 characters?

File: torch/onnx/symbolic.py [hunk 2]
```diff
@@ -1070,6 +1070,19 @@ def repeat(g, self, repeats):
...
+def pixel_shuffle(g, self, upscale_factor):
+    dims = self.type().sizes()
+    if len(dims) != 4:
+        return _unimplemented("pixel_shuffle", "only support 4d input")
+    output_channel = dims[1] // upscale_factor // upscale_factor
+    after_view = view(g, self, [-1, upscale_factor, upscale_factor,
+                                output_channel, dims[2], dims[3]])
+    after_transpose = g.op("Transpose", after_view, perm_i=[0, 1, 4, 2, 5, 3])
+    return view(g, after_transpose,
+                [-1, output_channel, dims[2] * upscale_factor, dims[3] * upscale_factor])
```
Comment (BIT-silence) [inline torch/onnx/symbolic.py]: ditto.

**Review by xiaomengy — APPROVED** [review]
